<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>517</width>
    <height>560</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Peer picker</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Peers:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTableView" name="peersTableView"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Find peer:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="searchPeerInput"/>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>(enter keywords for live search)</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Selected peer:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPlainTextEdit" name="selectedPeerText"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="buttonCall">
        <property name="text">
         <string>Call</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="buttonChat">
        <property name="text">
         <string>Chat</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>228</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="buttonAdd">
        <property name="text">
         <string>Add to Favorites</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Favorites:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTableView" name="favoritesTableView"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>517</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="actionQuit"/>
   </widget>
   <addaction name="menuMenu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionCall">
   <property name="text">
    <string>Call</string>
   </property>
   <property name="toolTip">
    <string>Call</string>
   </property>
  </action>
  <action name="actionChat">
   <property name="text">
    <string>Chat</string>
   </property>
  </action>
  <action name="actionAdd_to_favorites">
   <property name="text">
    <string>Add to favorites</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonCall</sender>
   <signal>clicked()</signal>
   <receiver>actionCall</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>44</x>
     <y>368</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonChat</sender>
   <signal>clicked()</signal>
   <receiver>actionChat</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>113</x>
     <y>368</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonAdd</sender>
   <signal>clicked()</signal>
   <receiver>actionAdd_to_favorites</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>436</x>
     <y>368</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
